src/atoms/inputField/InputField.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import PropTypes from 'prop-types';
+
+// Lodash
+import _noop from 'lodash/noop';
+
+//Styles
+import "./InputField.css"
+
+const InputField = ({inputType, todoText, handleTodoTextChange}) => {
```

Comment:
Now if you have bounded this with todo functionality (todoText), it is better to move this closer to todoForm.